transition_sampling/util/periodic_table.py

```diff
@@ -0,0 +1,43 @@
+"""Submodule that handles the generation of periodic table information."""
+from __future__ import annotations
+
+import os
+
+
+def atomic_symbols_to_mass(atoms: list[str]) -> list[float]:
+    """Converts atomic symbols to their atomic masses in amu.
+
+    Parameters
+    ----------
+    atoms
+        List of atomic symbols
+
+    Returns
+    -------
+    List of atomic masses"""
+    atomic_mass_dict = get_atomic_mass_dict()
+    masses = []
+    for atom in atoms:
+        masses.append(atomic_mass_dict[atom])
+    return masses
+
+
+def get_atomic_mass_dict() -> dict[str, float]:
```

Comment:
Something doesn't sit right with me about reading this file every time, I'd rather do it once at initialization. One way is wrapping these two methods in a class.

I think a better way is to make a submodule that has only `__init__.py` and in that init, it loads the file into a global (to that submodule) variable. This kinda follows the singleton pattern if you want to look more into it, but there should only ever be one of these dictionaries of masses so we can put in a global that never changes as a 'ground truth.'

Instead of `get_atomic_mass_dict()` it would instead be 
```from my_dict_handler import ATOMIC_MASSES```
and in the `my_dict_handler` `__init__.py`:
```
ATOMIC_MASSES = {}
...
```

or something like that

Reply:
This was addressed by adding an `atomic_masses` submodule in `util` which instantiates the `ATOMIC_MASSES` dictionary once and is now imported instead of rebuilt each time.